packages/amplify-cli/src/extensions/amplify-helpers/update-amplify-meta.ts

```diff
@@ -70,9 +73,9 @@ function moveBackendResourcesToCurrentCloudBackend(resources) {
   }
 }
 
-function removeNodeModulesDir(currentCloudBackendDir) {
+function removeNodeModulesDir(currentCloudBackendDirFunction) {
```

Comment:
Couple nitpicks:
I'd say leave the name as it was, because I first assumed you were passing a `function` based on the variable name, but I think you were trying to differentiate from layers.

Also, please add a type to the parameter - I'm assuming it's string, but it would be better if it's explicit

Reply:
good Catch! , Thanks